### ISSUE TITLE
Removing non-essential elements

### ISSUE THREAD

**Issue by SweetiePi** [SweetiePi/snacc#86]
Hey guys,

As I'm gearing SNACC up for publication, I'm going to be doing some spring cleaning on this repo. This will include:

- Removing useless features (sorry BWT!)
- Reducing the amount of package dependencies we use.
- Cleaning up directories & data from public view.
- General code refactoring.

I'll also be removing any non-essential features from SNACC. The focus here is that we want users to be able to easily install our pipeline without getting too overwhelmed with dependency issues and data.

**Comment by souravsingh** [comment]
Sounds good to me.

**Comment by wilcas** [comment]
No problems here. If you need help with refactoring stuff, especially stuff
that I wrote, I can definitely help, I just need to get a sense of where we
want to scale it back to (ie if we create issues it should be fine)

**Comment by wilcas** [comment]
Is tqdm "essential"?

**Comment by SweetiePi** [comment]
Not really, but I'd prefer to keep it as seeing the remaining time left is useful.

**Comment by Benjamin-Lee** [comment]
It's sufficiently lightweight and user-friendly that I'd think it makes sense to keep it (unless there's a compelling reason not to)

**Comment by wilcas** [comment]
The reason I would state is that it replaces a parallel map function with a
set of for loops and submits, which is likely to be far less efficient
